CMakelists.txt

```diff
@@ -0,0 +1,122 @@
+cmake_minimum_required(VERSION 3.0)
```

Comment:
I figured out why this file is not highlighted with CMake syntax. The name of the file is `CMakelists.txt` instead of `CMakeLists.txt` (upper-case L). Not a problem for Windows and OSX but on Linux it will produce error "The resource directory does not appear to contain CMakeLists.txt".

Reply:
Good catch.  I'm sending a new PR to address this.

Reply:
For the record https://github.com/hunter-packages/rabit/pull/2